inversify.config.ts

```diff
@@ -3,7 +3,7 @@ import TYPES from './ioc.types';
 
 // Config
 import getCloudDependencies from './source/Util/getCloudDependencies';
-import * as AWS from 'aws-sdk';
```

Comment:
There may be other places to make this change as well. This might reduce the size of the agent?